packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx

```diff
@@ -59,7 +59,7 @@ export const MarkdownPreview = ({ markdown, append, ...styleProps }: MarkdownPre
             {children}
           </Mention>
         ) : (
-          <a style={{ overflowWrap: 'break-word' }} href={href}>
+          <a style={{ overflowWrap: 'anywhere' }} href={href}>
```

Comment:
Good solution but please add it in the `MarkdownPreviewStyles` instead of inline.

Also could you try to add it at the `.markdown-preview *` level ? Because even though it's unlikely this issue would occur on anything else than a hyperlink it still might. Eg on a long inline code like this: `firstValueFrom(api.query.referendum.accountsOptedOut.keys()).then((result)`